tools/packaging/kernel/kata_config_version

```diff
@@ -1 +1 @@
-93
+94
```

Comment:
@fidencio is this bumped because of the config change or the version change ? `README.md` seems to indicate this gets bumped on config changes. Asking to be sure I get it right.

Reply:
It's bumped because of the configuration change.
We've added `CONFIG_SPECULATION_MITIGATIONS=y` as part of this PR.